### ISSUE TITLE
Phishing issue related to Augur?

### ISSUE THREAD

**Issue by AoDev** [kvhnuke/etherwallet#705]
I've got a message from a bot in Slack Augur (REP) channel. Saying they have to "update" the smart contract. They send a link to https://myetherwallet.xyz/fix-wallet.php.

This seems pretty much like a scam to me but I wanted to share it with you. I know Github is for development but I didn't know where to send this.

<img width="563" alt="screen shot 2017-07-07 at 11 39 29" src="https://user-images.githubusercontent.com/1526150/27952554-f94dc01c-6308-11e7-9724-a5d93707787a.png">

**Comment by AoDev** [comment]
One of the reasons that I mentioned this here, is that you have a big warning in red, on the website that "our domain" is `.com`. (which is good)

The problem is the link text here, is showing `.com` too. So it can quickly "disable" people's attention. Especially if the navigation bar is hidden for your browser.

**Comment by tayvano** [comment]
Just scroll down https://twitter.com/myetherwallet to see the shitstorm.

**Comment by AoDev** [comment]
I see.

I was wondering if on MyEtherWallet.com, the "best practices" / "must read" link should be closer to the interface CTA. 

For example:

![check-list](https://user-images.githubusercontent.com/1526150/27966788-cb250474-6340-11e7-8cc9-863e0af385c1.png)

**Comment by tayvano** [comment]
Good call. Will try to do at soem point soon.

We also have chrome extension now  Please install https://chrome.google.com/webstore/detail/etheraddresslookup/pdknmigbbbhmllnmgdfalmedcmcefdfn

**Comment by mikeyb** [comment]
Yeah this is tough.  At what point to you stop making big bold red text across your app?  I am a firm believer in tough love.  I think MEW does a fantastic job compared to most for warning its users, yet you still have people that fall for this crap.  At some point users are just going to have to accept responsibility for their actions.   I do think making it even more blatant on the first screen of MEW would help a bit more, but I feel like the only way to make it super obvious is to go the geocities route and cover the page in flashing gif animations or :parrots:, etc.

@AoDev are you part of the Augur team?  If not, can you put me in touch with someone who is?  I just dealt with this same sort of slackbot attack with the GAME/MGO slack server and want to compare notes.  Seeing this happening on quite a few slacks now and want to try and pinpoint the cause.  Im not sure if it was an API key that was compromised or an exploit in one of the slack apps.  I would really like to figure out how this is happening exactly and formulate a fix.  This is costing noobs quite a bit of money it seems.

Cheers

**Comment by tayvano** [comment]
Here are things we've learned

```
Hi Jack,

I completely understand your frustration here, and I can assure you that the team is taking this very seriously and actively investigating. I do not have a timeline I can share, however I have added you to the bug report we have for this and we will be sure to let you know once we have made any changes.

Right now, I'm afraid that restricting webhooks that pop up and deleting reminders is our best workaround, as well as letting your team know to avoid clicking on any links they do not recognize.

Thanks, and I am sorry I could not give you a better answer for this,

Simone
```

```
Hi Kevin,

Thanks for getting in touch and flagging this with us, I'm sorry for the hassle here.

To remove the reminder(s) from a public channel, enter /remind list which returns the list view of all the current reminders scheduled for said channel and the option to delete them. I'd suggest asking your Team Owner or Admin to enable the Admin Approved Apps feature which enables you to create a list of approved or restricted apps for their team.

At this time we don't have any other suggestions to prevent /remind abuse for communities, however, I'll let you know if we have any additional information.

I'm sorry I wasn't able to help further at this time, please let me know if you have any other questions.
```

```
Aubrey (Slack)
Jul 6, 1:14 PM PDT

Hi C,

Thanks for letting me know. An owner on the team can prevent users from creating tokens.

To find a list of team Owners on your team, go to the Account page: https://my.slack.com/account/team.

At the top, you'll see Owners and Admins listed. If you have several Administrators, you'll see a view admins link instead. Just click the link to see the complete list.

Those owners will need to know about Admin Approved Apps to get token creation restricted. That info can be found here:
https://get.slack.help/hc/en-us/articles/222386767-Manage-apps-for-your-team

Sorry for any trouble that you might be having here. I hope this helps.
```

```
Hi Jack,

Thanks for writing in. Thanks for the report and my apologies for this experience.

The team are looking into this, but we have a couple of workarounds for you in the meantime.

If you are seeing the phishing attempts as reminders in a channel, if you type /remind list you'll be able to view a list of the reminders in that channels, as well as an option to delete them.

Also I would also suggest that you as the PO enables Approved Apps so you can moderate better which custom integrations get added on your team.

Lastly of course, you can disable the accounts that may be sending this to your team.

I'm sorry that we can't be of more help at this time, Jack.

Please let me know if you have any questions.

Thanks,

Simone
```

```
I still think the best solution - for now - is to empower the users to be responsible for their own safety - they will need to know this in a digital future. (edited)

And that means teaching every single person in our community to learn a few things. (edited)

Main thing to learn:

We cannot stop anybody posting @slackbot reminders

We cannot delete @slackbot reminders in an IM channel that isn't ours (edited)

But - we can teach every single person who belongs to a slack channel how they can remove @slackbot reminders

- - - - - - - -

Every community member could learn this: (edited)

Go to your @slackbot channel - right there under DIRECT MESSAGES on the left hand side

In your @slackbot channel type this:

`/remind list`

that will then show up current and future reminders

Most likely these are all spam and you want to delete them

- - - - - - - - - -

Channel owners could: (edited)

Ensure nobody can assign themselves a token (check here: https://api.slack.com/custom-integrations/legacy-tokens) (edited)

Teach their users how to delete @slackbot IM messages / reminders using the above command (edited)

Maybe they could form a group of Slack channel owners who could share security ideas - I noticed some ICO teams had their tokens locked down and some did not. (edited)

Just thought of another thing - get people to switch off slackbot email notifications. Ok - I'll lay this up onto some kind of readme jpeg that we could put out to help people work around slackbot reminders.
```

https://medium.com/@altcoinio/slack-api-for-icos-de61df6448c3

**Comment by AoDev** [comment]
@mikeyb I am not part of Augur team. I was on their slack channel. I suggest you join their Slack team and you should find people to speak with. http://invite.augur.net/

Regarding the UI / UX warnings. I still think they would be better closer to the user actions. Like below the textarea where to copy paste the private key. For example right now, the red banner is not fixed on top, so it is out of view when scrolling. Plus, usually "small text" means "not so important to read". Could be just the annoying cookie warning plaguing every website on Internet.

**Comment by mikeyb** [comment]
For sure @AoDev but the real question is, how ugly do you want to make your application to cater to the less intelligent?  Historically evolution has wiped out the these types, but now we live in a world of helmets and knee pads so people don't hurt themselves.  At some point, catering to this lack of critical thinking causes an abundance of people who can't critically think (crypto is unimaginably full of this already).  To me, having a warning at all is overkill.  If I am using a currency whether it be physical or digital, I should have some understanding of what happens when I make a bad decision.

Such things like leaving my wallet (private keys) laying around is probably going to lead to a loss of funds.  Just like if I am hanging out at a local coffee shop and someone dressed as a barista walks up and says "Hey hand me $5 and I will go get you a new coffee so you dont have to get up".  To me it would make no sense why someone would ask me that, it is not a standard procedure.  Even if coffee shops were a new idea to me, I would probably ask my friend if this is for real.  Not just hand over $5.

People willing to click on a random link requesting them to move money around, and do so without questioning even the most basic things about the request, kinda deserve to lose their money IMO.  I do not expect a coffee shop to hang red banners over every table that say "Do not give money to strangers", this should be common sense derived from critical thinking.

Don't get me wrong, I am not advocating people getting scammed is a good thing.  User education is a much needed thing in crypto (literally the biggest issue about crypto imo.  Far worse than block size or tx throughput).  I don't really see it being fair to expect MEW to degrade the visual appeal of their wallet for these people though.  The giant red banner at the top is pretty damn obvious, and if you don't read it because you can't be bothered when dealing with your finances, then please by all means send your money to scammers so evolution will rid us of the flawed genetic pool.  Personally I find the red banner at the top annoying as all hell, but I understand it also being a small price to pay for a lot of user awareness.  But for the love of whatever you believe in, let us stop catering so much to the few who are lazy, ignorant, or just plain dumb at the expense of those who actually grasp basic concepts like people will steal your money if you don't pay attention in life and ask questions.

TL;DR:  Let's not ask MEW to add even more ugly red text and banners to cater to idiots.

**Comment by AoDev** [comment]
I am not asking for more red banners. Just trying to find the best place where to put these warnings.
Also it's not the same flow that many people are used to. No login, no 2-factor authentication, didn't have to identify yourself. The whole flow is just write some random text in an input and here you go, address and all. I wouldn't consider people idiots for mistakes.
MEW has a self-signed certificate. For years people were mislead to look at the "secure" message in their address bar to decide whether they are on a secure site or not. I wouldn't talk so high and mighty about people who, in the end, show interest in such technologies and are learning. Usually you fall before you learn, calling them idiots before they fall because they didn't look well and even say they deserve it, is just mean.

**Comment by tayvano** [comment]
>  No login, no 2-factor authentication, didn't have to identify yourself.

😩

**Comment by tayvano** [comment]
IF you want to have those features, please leave your crypto on and exchange. Thansk

**Comment by AoDev** [comment]
I think you misunderstood my writings. I was trying to explain why such a different flow may look strange to newcomers. 

EDIT: hence the possible mistakes from beginners. So I don't think it's nice to call them idiots. Personally I don't have an issue with it although the flow is still strange. For example having to copy paste a private key on a website is something I thought was going against any security warnings I've read until now.

**Comment by mikeyb** [comment]
Being a new comer does not excuse you from learning what you are interacting with.  You dont just get a drivers license, you learn the rules and how things work, then you take a test.

Lacking intelligence does not make someone an idiot.  Proceeding with financial distribution before understanding something does.

I still stand firmly that people need to educate themselves, expecting a software developer to change the way they designed their application UI/UX for idiots is absurd.  I don't even think these devs get income from this project?

> No login

Do you login to the wallet in your back pocket?

> No 2-factor authentication

Why would an offline utility require online validation? 

> Didn't have to identify yourself.

See above and also add, this is crypto.  Your private key is a string of text, who are you suppose to identify to?  I don't identify myself when I hand someone a $20 bill

I am literally confused why this would be expected of free software on a financial network.  If people can't be bothered to learn something, they deserve to get beat down by the system they are using.  Just like I would expect to crash into a wall and die if I got into a car without understanding how to use it.

People will be idiots.  People will lose money.  That is how you learn in life and become less idiotic.  Sure, people make mistakes, but anyone sending money to a scammer from the scam you posted deserves it.  They proved to be either lazy, ignorant, or just plain dumb.  Losing money will hopefully teach them the lesson of what being lazy, ignorant or dumb results in.  Again, I am not advocating scams.  But if you can't be bothered to learn something (or in the very least ask for help), especially something involving your finances, seriously go fuck yourself and jump in a pit.  I mean this in the truest sense of the words.

Anyone scammed could have easily not been scammed by performing this one weird trick:

*Ask Questions*

RE: Your high and mighty comment.  You got the wrong guy man.  If you thought I was coming off as high and mighty check your head.  You know how I learned all this stuff?  By reading and asking questions.  Do you know how many scammers scam me?  0

Please by all means, fork the software and make it all red with 19 confirmations that tell people to verify blah blah blah.  I can assure you with great certainty there will still be an idiot who loses money.  Just like the person wearing a helmet and knee pads can still ride their bike off a mountain.

Sure, MEW can move the warnings.  They could make them bigger.  Make them more red.  Make them flash.  You will still have someone sending money to a scammer because their ego was too big to be humble and ask a question.  So incredibly sad humans are this idiotic, even when they have the most powerful tool at their fingertips.  Please, let evolution thin the herd.

**Comment by mikeyb** [comment]
And just to be clear, I think your points and suggestions are quite valid.  But at what point do we stop catering?  At what point is a big red banner at the top of the page not enough?  If you skip past that, and you are new to this stuff, this is where you become the idiot I speak of.

**Comment by AoDev** [comment]
I don't know why you insist on the fact the website should be turned into an ugly state, with exaggerated warnings. Surely a UI/UX person can make a good job at designing a nice UI and carefully prepare a flow that can help people think better about what they are doing.

Now I won't continue this discussion. I've opened this Github issue to warn about the current phishing issue and suggest helping people by maybe, improving the current place of the warnings. 
Good luck.

**Comment by AoDev** [comment]
Also, since I am not a user of reddit but on this thread: https://www.reddit.com/r/ethereum/comments/6lfy73/warning_stop_clicking_links_stop_sending_to/ I've wanted to add the .xyz domain to the urls to report. But I have no reputation so it has been moderated. If one of you want to add.

**Comment by mikeyb** [comment]
> Surely a UI/UX person can make a good job at designing a nice UI

And to you I say Fork the repo and make the world the way you see it working best.  Like I said.  I don't think they get paid.  If they do get some amount, I am sure it covers nothing close to the time costs they have submitted.

How about people just wise up and *learn* to use the tools people are offering them for *free*.

**Comment by tayvano** [comment]
Ive got a bunch of URLs and Slacks and addresses and stuff here: https://docs.google.com/spreadsheets/d/1ErQGI2elbzVAapLBYzDePV7jqpiDnsJoSlmAlQ9_zno/edit?usp=sharing

I literally don't have time to do it all. If you want something, open a PR, please. Seriously. Please.
